charts/quantum-serverless/values-ibm.yaml

```diff
@@ -34,17 +34,7 @@ ingress:
 
```

Comment:
Now that we finished with the integration with the infrastructure team I think we could remove this file.

Reply:
OK.  I remove it.  Thanks!